src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1015,7 +1015,7 @@ class DocBaseViewer extends BaseViewer {
         this.controls = new Controls(this.containerEl);
         this.pageControls = new PageControls(this.controls, this.docEl);
         this.zoomControls = new ZoomControls(this.controls);
-        if (this.options.showAnnotationsControls) {
+        if (this.areNewAnnotationsEnabled()) {
             this.annotationControls = new AnnotationControls(this.controls);
```

Comment:
This disables the controls, but doesn't disable the Annotations SDK, so annotations that great created in a different way (platform, API, etc.) will still be shown. Are we concerned about that scenario?

Reply:
> This disables the controls, but doesn't disable the Annotations SDK, so annotations that great created in a different way (platform, API, etc.) will still be shown. Are we concerned about that scenario?

Yeah, Rohit said we only want to disable new annotations for these extensions.

Reply:
Does that mean "newly-created" or "new experience"? I suspect that he meant the former. Can you check?

Reply:
> Does that mean "newly-created" or "new experience"? I suspect that he meant the latter. Can you check?

Oh I realized this difference just now...